src/app/components/Home/ListOfAddresses/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import List from "@material-ui/core/List";
+import ListItem, { ListItemProps } from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Divider from "@material-ui/core/Divider";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import { ListItemIcon } from "@material-ui/core";
+
+const ListItemLink = (props: ListItemProps<"a", { button?: true }>) => {
+  return (
+    <>
+      <li>
+        <ListItem button component="a" {...props} />
```

Comment:
`button` or `a`?

Reply:
It is- a. 
I used material-ui component https://codesandbox.io/s/ogfu9?file=/demo.tsx

![image](https://user-images.githubusercontent.com/36860438/127979746-b3c94bab-b507-4a7a-929d-153d05509910.png)

Reply:
Ok. I was curious if both are required (`button` and `a`)